tutoraspects/templates/aspects/apps/superset/pythonpath/openedx_sso_security_manager.py

```diff
@@ -209,7 +230,9 @@ def get_courses(self, username, permission="staff", next_url=None):
             username=username, permission=permission
         )
         url = next_url or courses_url
-        response = oauth_remote.get(url, token=token).json()
+        resp = oauth_remote.get(url, token=token)
+        resp.raise_for_status()
```

Comment:
Previously we ignored errors here which limited logging things like the LMS 429 rate limit errors, this will now raise them for easier debugging.